### PR TITLE
Add runtime related changes to beta 2 release note

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
$Subject

### CODE REVIEW

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-beta2-release-note.md [hunk 1]
```diff
@@ -36,6 +36,27 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+The `Configurable` feature is improved to support variables with the `union` types through the TOML syntax.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta2-release-note.md:39]: This needs to be a bullet point, right?

File: release-notes/swan-lake-beta2-release-note.md [hunk 2]
```diff
@@ -36,6 +36,27 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+The `Configurable` feature is improved to support variables with the `union` types through the TOML syntax. 
+
+Here are some examples with `union` types.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta2-release-note.md:41]: Do we need this? We've just added the examples for others IINM.

File: release-notes/swan-lake-beta2-release-note.md (hunk 1 above)
Comment (warunalakshitha) [reply release-notes/swan-lake-beta2-release-note.md:39]: This is same as beta 1 release note. Do We need to change?

File: release-notes/swan-lake-beta2-release-note.md (hunk 2 above)
Comment (warunalakshitha) [reply release-notes/swan-lake-beta2-release-note.md:41]: This is also the same as beta 1 release note when we support enums.

File: release-notes/swan-lake-beta2-release-note.md (hunk 1 above)
Comment (MaryamZi) [reply release-notes/swan-lake-beta2-release-note.md:39]: This breaks the flow with the rest of the points, please check https://github.com/ballerina-platform/ballerina-release/blob/master/release-notes/swan-lake-beta2-release-note.md#improvements. Currently looks like it belongs to "Support for recursive tuple types has been introduced with this release.".

We either use bullet points or subheadings. Here, I guess we can go with bullet points.

File: release-notes/swan-lake-beta2-release-note.md (hunk 2 above)
Comment (MaryamZi) [reply release-notes/swan-lake-beta2-release-note.md:41]: I guess it is okay, but we haven't said anything like this for the rest. Was wondering if it is inconsistent.

File: release-notes/swan-lake-beta2-release-note.md (hunk 1 above)
Comment (warunalakshitha) [reply release-notes/swan-lake-beta2-release-note.md:39]: will send a fix in a separate PR

**Pull request merged**